### PR TITLE
Add resize function to inference_style_transfer.py & Fix one bug

### PR DESCRIPTION
**Pull request by uowol**
### Add resize function to inference_style_transfer.py
- At `inference_style_transfer.py`, an error occurs if the resolution of content_image is different from the resolution of the image used during training. Therefore, added the resize function to `inference_style_transfer.py`.

### Fix one bug
- At `inference_t2i.py`, an error occurs by parentheses problems. Solved the error.

### CODE REVIEW

File: dreamstyler/inference_style_transfer.py [hunk 1]
```diff
@@ -82,18 +85,22 @@ def style_transfer(
     cross_attention_kwargs = {"num_stages": num_stages}
 
     content_image = Image.open(content_image_path)
+    content_image = content_image.resize((resolution, resolution))
     control_image = processor(content_image, to_pil=True)
     pos_prompt = [prompt.format(f"{placeholder_token}-T{t}") for t in range(num_stages)]
 
     outputs = []
-    for _ in range(num_samples):
+    # torch.manual_seed(1)
```
Comment (nmhkahn) [inline dreamstyler/inference_style_transfer.py:93]: Is there any specific reason to change num_samples to one? if not, could you rollback this?

**Review by nmhkahn — COMMENTED** [review]
I appreciate your contribution. It looks good but I have a single minor question before approve this.

File: dreamstyler/inference_style_transfer.py (hunk 1 above)
Comment (uowol) [reply dreamstyler/inference_style_transfer.py:93]: Oh, I modified it for a quick experiment, but I didn't restore it. This is my mistake. I'll rollback it :D

Comment (uowol) [reply dreamstyler/inference_style_transfer.py:93]: done. 😊

**Pull request merged**